tests/test-class-module.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+class WP_Test_Edit_Flow_Class_Module extends WP_UnitTestCase {
+	protected static $admin_user_id;
+
+	public static function wpSetUpBeforeClass( $factory ) {
+		self::$admin_user_id = $factory->user->create( array( 'role' => 'administrator' ) );
+	}
+
+	function _flush_roles() {
+		// we want to make sure we're testing against the db, not just in-memory data
+		// this will flush everything and reload it from the db
+		unset($GLOBALS['wp_user_roles']);
+		global $wp_roles;
+		if ( is_object( $wp_roles ) )
+			$wp_roles->_init();
+	}
+
+	function setUp() {
+		parent::setUp();
+		// keep track of users we create
+		$this->_flush_roles();
+
+	}
+	
+	function test_add_caps_to_role() {
+		$EditFlowModule = new EF_Module();
+
+		$usergroup_roles = array(
+			'administrator' => array( 'edit_usergroups' ),
+		);
+
+		foreach( $usergroup_roles as $role => $caps ) {
+			$EditFlowModule->add_caps_to_role( $role, $caps );
+		}
+
+		$this->_flush_roles();
```

Comment:
Yea, was wondering about that. In the `test_role_add_cap` function [capabilities.php](https://develop.svn.wordpress.org/tags/4.5/tests/phpunit/tests/user/capabilities.php) a call to `_flush_roles` appears after every call to `add_cap`.

Do you know if there's a reason for that? That's why I added that `_flush_roles` call in the first place